app/services/facilities/website_url_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Facilities
+  class WebsiteUrlService
+    FACILITY_TYPES = {
```

Comment:
This is no longer needed, right?

Reply:
Correct, it got re-added by accident. Thanks @jholton!